Rem/Bot/DiscordBot.cs

```diff
@@ -1,14 +1,12 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Diagnostics;
 using System.Reflection;
-using System.Text;
 using System.Threading.Tasks;
 using Discord;
 using Discord.Commands;
 using Discord.WebSocket;
 using Microsoft.Extensions.DependencyInjection;
 using SQLite;
+using Image = Rem.Models.Image;
```

Comment:
Don't do this. It makes it unclear in code if we are referring to an actual image (as in SixLabors) or to the model itself